handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -370,6 +370,13 @@ public List getRequestedServerNames() {
                     }
                 }
 
+                if (OpenSsl.isBoringSSL() && clientMode) {
+                    // If in client-mode and BoringSSL let's allow to renegotiate once as the server may use this
+                    // for client auth.
+                    //
+                    // See https://github.com/netty/netty/issues/11529
+                    SSL.setRenegotiateMode(ssl, SSL.SSL_RENEGOTIATE_ONCE);
```

Comment:
Do we just want to always set this to `_ONCE` or would it be better to make this configurable?
SwiftNIO made it configurable, and I don't see a rationale for why we wouldn't do the same.

Reply:
Is this perhaps related to the `SSL.getHandshakeCount(ssl) > 1` check in `rejectRemoteInitiatedRenegotiation()`?

Reply:
I just thought `_ONCE` should be good enough for the client auth while still add some sort of protection. OpenSSL always just allows it and the JDK as well. SwiftNIO only supports BoringSSL so its a bit easier for them .

Reply:
Ok, fair enough